### PR TITLE
fix: Update colors for NetworkIndicatorView

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we update the colors for the `NetworkIndicatorView`.  In addition to that, we get rid of `override var preferredStatusBarStyle` from these classes since it's not needed anymore. 

- CollectionsViewController
- GiphyNavigationController
- ConversationContentViewController
- MessageDetailsViewController
- FolderCreationController
- FolderPickerViewController
- GroupDetailsViewController
- SettingsClientViewController
- UserClientListViewController
- ProfileViewController

Last but not least we get rid of two properties in ColorScheme+Helper.swift since they're not used anymore.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift [hunk 1]
```diff
@@ -46,7 +46,7 @@ final class NetworkConditionIndicatorView: UIView, RoundedViewProtocol {
             label.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10),
             label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10)
         ])
-        backgroundColor = UIColor.nameColor(for: .brightOrange, variant: .light)
+        backgroundColor = overrideUserInterfaceStyle == .dark ? UIColor.accentDarken : UIColor.accent()
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: The `accent color` is a `semantic` color. We should not check interfaceStyle manually.

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift [hunk 2]
```diff
@@ -37,7 +37,7 @@ final class NetworkConditionIndicatorView: UIView, RoundedViewProtocol {
         isAccessibilityElement = true
         shouldGroupAccessibilityChildren = true
 
-        backgroundColor = UIColor.nameColor(for: .brightOrange, variant: .light)
+        backgroundColor = overrideUserInterfaceStyle == .dark ? UIColor.accentDarken : UIColor.accent()
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: the same comment is below

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift (hunk 1 above)
Comment (KaterinaWire) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: Why was the color always orange before? Are you sure we should use an accent color?

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: I checked the designs, it should be accent color, yes.

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift (hunk 2 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: Done!

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift]: Done!

**Review by KaterinaWire — APPROVED** [review]

**Review by netbe — APPROVED** [review]

**Pull request merged**